src/plugins/intel_gpu/src/plugin/ops/loop.cpp

```diff
@@ -97,6 +97,19 @@ static void SetLoopInputOutputMap(ProgramBuilder& p,
             cldnn::primitive_id from_id = layer_type_name_ID(from);
 
             back_edges_maps.emplace_back(from_id, to_id);
+            // tmp solution
```

Comment:
Maybe remove the tmp solution code if not needed.

Reply:
please remove this temp codes